pluginsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -102,6 +102,8 @@ namespace Microsoft.Windows.DevHome.SDK
 
         IPluginAdaptiveCardController GetAdaptiveCardController(String[] args);
 
+        Windows.Foundation.IAsyncOperation<IDeveloperId> ShowLogonSession(Microsoft.UI.WindowId windowHandle);
```

Comment:
Can we use `ulong` instead (or equivalent)? This allows us to decouple Dev Home SDK from the UI technology used.
```cs
ulong input = 123;
var wId = new Microsoft.UI.WindowId(input);
wId.Value.ToString(); // 123
```

Bonus: With this approach, we may not need to depend (and continuously align) the WindowsAppSDK nuget

Reply:
I second this.  Mostly to remove a dependency on Windows App SDK.

Reply:
During initial API design review it was recommended that we use the WindowId type instead of IntPtr type (IntPtr was in the initial design for the interface). I believe this recommendation would be preferred over ulong as well. We look furhter today at the meeting. Thanks!